.github/workflows/gitpod.yml

```diff
@@ -0,0 +1,36 @@
+---
+name: "Gitpod"
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
```

Comment:
We use main branch, not master ^^

Reply:
my bad, fix here :
https://github.com/oxsecurity/megalinter/pull/2737/commits/e4fb2053eaf896aebf54e33a67ce6eabce369fbb